dataland-frontend/tests/e2e/fixtures/common/NumberFixtures.ts

```diff
@@ -1,36 +1,30 @@
 import { faker } from "@faker-js/faker";
 
-const percentagePrecision = 0.0001;
-const maxEuro = 1000000;
-const minEuro = 50000;
-
 /**
  * Generates a random decimal value in [min, max]
  * @param min the minimum allowed value (inclusive)
  * @param max the maximum allowed value (inclusive)
+ * @param precision the precision of the decimal value
  * @returns a random number in [min, max]
  */
-export function randomEuroValue(min: number = minEuro, max: number = maxEuro): number {
-  return faker.number.float({ min: min, max: max });
+export function randomFloat(min: number, max: number, precision?: number): number {
+  return faker.number.float({ min: min, max: max, precision: precision });
 }
 
 /**
- * Generates a random number in [0, max]
- * @param max the maximum allowed value (inclusive)
- * @returns a random number in [0, max]
+ * Generates a random decimal value between 0 and 1
+ * @param precision the precision of the decimal value
+ * @returns a random float between 0 and 1
  */
-export function randomNumber(max: number): number {
-  return faker.number.int(max);
+export function randomPercentageValue(precision = 0.0001): number {
+  return randomFloat(0, 1, precision);
```

Comment:
Aren't we storing percentages as values between 0 and 100?

Reply:
Additionally, there is overlap with the function `generatePercentage()`, or not?

Reply:
There is a story to clean up percentage usage.